packages/web-pkg/src/event/bus.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { EventBus } from './bus'
```

Comment:
File is not in the right location (needs to be in the `tests/unit` folder hierarchy)